pkg/env/armhelper.go

```diff
@@ -0,0 +1,105 @@
+package env
```

Comment:
this commit got messed up and the feature flag is lost and the FPClientID changes seem to have got inserted

Reply:
documentation needed for armhelper.